src/libYARP_dev/src/devices/RemoteControlBoard/RemoteControlBoard.cpp

```diff
@@ -3749,6 +3719,21 @@ class yarp::dev::RemoteControlBoard :
         extendedPortMutex.post();
         return ret;
     }
+
+#ifndef YARP_NO_DEPRECATED // since YARP 2.3.70
+#if !defined(_MSC_VER)
+YARP_WARNING_PUSH
+YARP_DISABLE_DEPRECATED_WARNING
+#endif
+    YARP_DEPRECATED bool setPositionMode(int j) { return setControlMode(j,VOCAB_CM_POSITION); }
+    YARP_DEPRECATED bool setVelocityMode(int j) { return setControlMode(j,VOCAB_CM_VELOCITY); }
+    YARP_DEPRECATED bool setTorqueMode(int j) { return setControlMode(j,VOCAB_CM_TORQUE); }
+    YARP_DEPRECATED bool setImpedancePositionMode(int j) { return setControlMode(j,VOCAB_CM_IMPEDANCE_POS); }
+    YARP_DEPRECATED bool setImpedanceVelocityMode(int j) { return setControlMode(j,VOCAB_CM_IMPEDANCE_VEL); }
+#if !defined(_MSC_VER)
+YARP_WARNING_POP
```

Comment:
Notice how this is different from [current master](https://github.com/robotology/yarp/blob/3e63e13/src/libYARP_dev/src/devices/RemoteControlBoard/RemoteControlBoard.cpp#L4140). I guess it was a typo (c5b8f28).